debezium-connector-binlog/src/test/java/io/debezium/connector/binlog/BinlogSnapshotSourceIT.java

```diff
@@ -155,7 +154,7 @@ private void snapshotOfSingleDatabase(boolean useGlobalLock, boolean storeOnlyCa
                     .with(BinlogConnectorConfig.USER, "cloud")
```

Comment:
Modification in the test is required due to the conditional behavior of `useGlobalLock` here. Initially `STORE_ONLY_CAPTURED_TABLES_DDL` was set as true by default irrespective of the lock but now only this condition sets the config option.